better nominate

exd02 opened the pull request:
+ Find the map with string containing the given name
+ Added translation (with ChatGPT) for the case of multiple maps with the same given name

*Example: Before, the `!nominate swag` command didn't return any maps
![Captura de tela 2024-02-27 002124](https://github.com/abnerfs/cs2-rockthevote/assets/62575526/0f7658d8-e72c-48e5-b348-ef52b92ec863)

exd02 committed:
find a map containing the given name instead of selection directly

exd02 committed:
moving the method to respective class

abnerfs commented:
Thanks for the contribution, about changing the folder structure I see no value, I have a github action that generates the build automatically when I upload a new tag, so build it to bin is working fine, I also don't like having the entire folder structure from addons in the zip, just the RockTheVote folder is enough.
About the nomination search it seems interesting, I will take a closer look in your implementation soon before we merge it, note that adding a new phrase to the translation means you need to add it to all languages the plugin supports not only to en and pt-br https://github.com/abnerfs/cs2-rockthevote?tab=readme-ov-file#translations so we also need to get it translated before merging

exd02 committed:
Added new phrase

exd02 commented:
- Removed folder structure thing
- Added translation to all languages (using ChatGPT)

Tested again:
![image](https://github.com/abnerfs/cs2-rockthevote/assets/62575526/52f9b3b4-104f-44cf-bc3c-fb771ba7b3a6)

exd02 commented:
It's causing error when voting for a map with a similar name (I have mg_minecraft and mg_minecraft_aphixed):
![image](https://github.com/abnerfs/cs2-rockthevote/assets/62575526/9826d543-166c-4a85-9e7a-4639f036e1a5)

exd02 committed:
select a map that is equals to the string before searching for maps that contain the string

exd02 committed:
Merge branch 'main' of https://github.com/exd02/cs2-rockthevote

exd02 commented:
fixed

Oylneechan commented:
Suggestion: Make a menu for in case there are more than 2 maps that contain the same name. If this is not in your plan now, maybe we could add it later.

Oz-Lin commented:
> Suggestion: Make a menu for in case there are more than 2 maps that contain the same name. If this is not in your plan now, maybe we could add it later.

Will work on this as well if I have time

Oz-Lin commented:
https://github.com/Oz-Lin/cs2-rockthevote/commit/1c19a3b4fe31a177eab948902286e3d8d11f0497
Listed in chat when multiple maps that contain the same name for now
